app/editor/src/features/content/form/components/upload/Upload.tsx

```diff
@@ -57,26 +57,44 @@ export const Upload: React.FC<IUploadProps> = ({
   stream,
   ...rest
 }) => {
+  const [loading, setLoading] = React.useState(true);
   const { isShowing, toggle } = useModal();
   const { values, setFieldValue } = useFormikContext<IContentForm>();
   const [, { ffmpeg }] = useWorkOrders();
   const { isShowing: showFFmpegModal, toggle: toggleFFmpeg } = useModal();
 
-  const [file, setFile] = React.useState<IFile>();
+  const [file, setFile] = React.useState<IFile | undefined>();
```

Comment:
Not required as typescript will know it can be undefined in this case because it's not set.  Same with below `useState()` calls.